kentikapi/credentials.go

```diff
@@ -40,39 +64,32 @@ func ReadCredentialsFromProfile() (authEmail, authToken string, err error) {
 	if err != nil {
 		return "", "", err
 	}
-	for _, s := range []string{"email", "api-key"} {
-		if _, ok := result[s]; !ok {
-			return "", "", fmt.Errorf("no '%s' in config file: %s", s, filename)
-		}
+	if result.AuthEmail == "" {
+		return "", "", fmt.Errorf("no AuthEmail in config file: %s", filename)
 	}
-	return result["email"], result["api-key"], nil
+	if result.AuthToken == "" {
+		return "", "", fmt.Errorf("no AuthToken in config file: %s", filename)
+	}
+	log.Printf("Credentials loaded from %v\n", filename)
+	return result.AuthEmail, result.AuthToken, nil
 }
 
-func getFilename(profile string) string {
+func resolveHome() string {
```

Comment:
We have a practice in Go codebase that helper functions are below their usages. So please move resolveHome below getFilename

Reply:
Done